packages/MSXIRC.yaml

```
@@ -0,0 +1,120 @@
+---
+name: 'MSXIRC'
+version: '1.1'
+release: 1
+summary: 'MSXDOS IRC client'
+author: 'Ptero and Oduvaldo Pavan Junior'
+package_author: 'Fubukimaru'
+license: 'Unknown'
+category: 'Internet'
+system: 'MSX'
```

I read in the documentation this requires at least MSX2, so this should probably be changed, right?
```suggestion
system: 'MSX2'
```

The list of systems can be found here: https://msxhub.com/list_systems